tests/test_driver_local_modules.py

```diff
@@ -0,0 +1,7 @@
+from hamilton.driver import Builder, DefaultGraphExecutor
+
+
+def test_driver_with_local_modules() -> None:
+    dr = Builder().with_local_modules().build()
+    assert isinstance(dr.graph_executor, DefaultGraphExecutor)
+    assert __name__ == list(dr.graph_modules)[0].__name__

```

Comment:
I think this test probably needs to pull in a function and exercise the driver.

e.g. define 
```python
def a(b:int) -> int:
   return b * 2
```

Then we should execute the driver to get "a". 

That should work right?